cliquet/resource.py

```diff
@@ -753,6 +755,15 @@ def _add_timestamp_header(self, response, timestamp=None):
         # Return timestamp as ETag.
         response.headers['ETag'] = encode_header('"%s"' % timestamp)
 
+    def _add_cache_header(self, response):
+        """Add Cache-Control and Expire headers, based a setting for this
+        resource.
+        """
+        key = 'cliquet.%s_cache_expires_seconds' % self.context.resource_name
```

Comment:
I thought of this but gave up with the idea. Do you have a strong argument in favor of distinguishing collection and records ?